dataland-frontend/tests/component/utils/SearchCompaniesForFrameworkDataPageDataRequester.cy.ts

```diff
@@ -1,80 +0,0 @@
-import { QaStatus, type DataTypeEnum, type StoredCompany, IdentifierType } from "@clients/backend";
-import type Keycloak from "keycloak-js";
-import { getCompanyDataForFrameworkDataSearchPage } from "@/utils/SearchCompaniesForFrameworkDataPageDataRequester";
```

Comment:
Why is this test no longer necessary?

Reply:
Because the removing of companies with non accepted datasets only is now done already in the queue

Reply:
I added a new backend e2e test to test for this.

Reply:
ok